.github/workflows/module-publish.yml

```diff
@@ -14,14 +14,18 @@ jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v1
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
```

Comment:
we need the full commit history, otherwise `lerna version` doesn't work properly